src/fragments/lib/react-native-polyfills.mdx

```diff
@@ -0,0 +1,7 @@
+### Install Amplify and its dependencies
```

Comment:
There's actually not really a difference between the In-App and the Interactions set of instructions. On Line 10 of the Interactions polyfill fragment, there's a mention of Interactions but it isn't actually used by anything here. I think we can extract the below fragments and consolidate into this file and just share them between the two categories

src/fragments/lib/in-app-messaging/integrate-your-application/react-native/polyfills-index.mdx
src/fragments/lib/interactions/js/frontend.mdx
src/fragments/lib/interactions/react-native/polyfills-index.mdx

Reply:
sounds good to me

Reply:
kept the `frontend.mdx` file because the interactions page is set up differently in JS and the other fragment wouldn't make sense with the headings

Reply:
Ok that makes sense if that fragment is intended for JS only